src/DB/Table/AdsRecommendationsTable.php

```diff
@@ -0,0 +1,69 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\DB\Table;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\DB\Table;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class AdsRecommendationsTable
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\DB\Tables
+ */
+class AdsRecommendationsTable extends Table {
+
+	/**
+	 * Get the schema for the DB.
+	 *
+	 * This should be a SQL string for creating the DB table.
+	 *
+	 * @return string
+	 */
+	protected function get_install_query(): string {
+		return "
+CREATE TABLE `{$this->get_sql_safe_name()}` (
+	recommendation_id bigint(20) NOT NULL AUTO_INCREMENT,
```

Comment:
@mukeshpanchal27 I believe we are meant to create the recommendation ID from the resource name so this shouldn't auto increment.

This is from the implementation brief.

> The `recommendation_id` is created by parsing and extracting it from the resource_name. The `resource_name` follows a format of `customers/{customer_id}/recommendations/{recommendation_id}`.

Reply:
I miss read that and missed totally. Thanks for flagging